src/js/apps/globals/app-frame_app.js

```diff
@@ -63,7 +63,7 @@ export default App.extend({
       }
     });
   },
-  onBeforeStop() {
+  onStop() {
```

Comment:
This doesn't appear to fix the issue where the nav menu shows on form page refreshes.

Reply:
if we keep this `stop` and add `if (this.isRunning()) return` to the top of `toggleNav` that should do it.

Reply:
Like this?

```js
  toggleNav(shouldShow) {
    if (this.isRunning()) return;

    this.getView().toggleNav(!!shouldShow);
  },
```

That makes it so the nav menu is always showing.

Reply:
`!this.isRunning()` seems to have fixed the issue